tests/Integration/Momento.Sdk.Tests/Utils.cs

```diff
@@ -11,4 +11,12 @@ public static class Utils
     public static string NewGuidString() => Guid.NewGuid().ToString();
 
     public static byte[] NewGuidByteArray() => Guid.NewGuid().ToByteArray();
+
+    public static int initialRefreshTtl = 4;
+
+    public static int updatedRefreshTtl = 10;
+
+    public static int waitForItemToBeSet = 100;
+
+    public static int waitForInitialItemToExpire = 4900;

```

Comment:
Style nit: use properties here. I recommend doing the rename first then adding the property getter.

```suggestion
    public static readonly int InitialRefreshTtl { get; } = 4;

    public static readonly int UpdatedRefreshTtl { get; } = 10;

    public static readonly int WaitForItemToBeSet { get; } = 100;

    public static readonly int WaitForInitialItemToExpire { get; } = 4900;

```

Reply:
👍 , apparently just adding `{ get; }` makes the entire property readonly, and if you try and set it it like `Utils.InitialRefreshTtl = 1111` its a compile time error.